cloud/stacks/index.md

```diff
@@ -19,8 +19,7 @@ synchronize a state of all stacks when orchestrated with Terramate CLI and keep
 deployments or drift runs.
 
 ::: tip
-The state of a stack is not to be confused with a Terraform state defined by the Terraform Backend Configuration.
-Still, the Terramate Cloud State of a stack includes this information and extends it with multiple status values and metadata.
+Do not confuse the state of a stack with the Terraform state defined by the Terraform Backend Configuration. The Terramate Cloud state of a stack includes the Terraform state and extends it with additional status values and metadata.
```

Comment:
is `state` the right term here? we sync "stack details" + its "status".

Reply:
Because the Terramate Cloud details for a particular stack is a superset of Terraform state, I included the word 'state' with Terramate Cloud too. Let me know if we can use a better word to avoid any confusion.

Reply:
I just added a commit that phrases this a lot better

Reply:
hope that makes sense